vald/values.yaml

```diff
@@ -269,7 +270,9 @@ gateway:
     discoverer:
       duration: "2s"
       discover_client: {}
-      agent_client: {}
+      agent_client: {
```

Comment:
![badge](https://img.shields.io/badge/review-suggestion-blue.svg?style=flat-square)
I think it doesn't needed to use "{}" here.

Reply:
I think we need, we have to control agent_client's connection pool size

Reply:
my sugggestion is just like this:
```suggestion
      agent_client: 
         connection_pool: 100
```

Reply:
I think we doesn't need L273's { and L275's }

Reply:
Yes.

Reply:
yes, I fixed like that

Reply:
thankyou